### ISSUE TITLE
tiles: a small error when loading 3dtiles.

### ISSUE THREAD

**Issue by Wincy94** [visgl/loaders.gl#1202]
When I try to load my 3dtileset data, it throws out an error:

`viewerRequestVolume.distanceToCamera is not a function;`

In @loaders.gl/tiles tile-3d.js line364
I suppose to add a judgement that can fix this error will be best

`return !viewerRequestVolume || !viewerRequestVolume.distanceToCamera ||viewerRequestVolume.distanceToCamera(frameState) === 0.0;`

**Comment by ibgreen** [comment]
We can certainly make that fix though I suspect the root cause might be deeper. Are you able to share a tileset that triggers this?

@Avnerus @dryabinin-actionengine FYI

**Comment by Wincy94** [comment]
> We can certainly make that fix though I suspect the root cause might be deeper. Are you able to share a tileset that triggers this?
> 
> @Avnerus @dryabinin-actionengine FYI

I'm sorry... For confidentiality purposes, I cannot share this tile tileset. No other problems have occurred at this time. I will continue to use this framework, and if there are other questions, I will report them in time.

**Comment by Avnerus** [comment]
Hi @ibgreen @dryabinin-actionengine @Wincy94,
this seems to be related to Cesium's [request volume feature](https://github.com/CesiumGS/3d-tiles-samples/tree/master/tilesets/TilesetWithRequestVolume) which I don't currently have a working example for. However that line does seem wrong because the math.gl culling objects do not have a `distanceToCamera` function.
The line should probably be replaced with
```
return !viewerRequestVolume || 
viewerRequestVolume.distanceTo(frameState.camera.position) === 0.0;
```
But I can't fully test this yet.

/Avner

**Comment by ibgreen** [comment]
@Wincy94 are you able to test @Avnerus suggested fix?

**Comment by ibgreen** [comment]
@Avnerus We should also make sure typescript checking is enabled for the 3d-tiles module this seems like something that could be caught automatically.

**Comment by Wincy94** [comment]
@Avnerus I tried this fix. It doesn't throw any errors, but the tileset doesn't show either.

**Comment by dryabinin-actionengine** [comment]
@Wincy94 Fix was published in `2.3.13` release.
